services/webhook/tests/test_code_reviewer_dispatch.py

```
@@ -2187,3 +2187,176 @@ def _fake_refute(findings, hunks, installation_id, **kw):
+    assert rec["author_login"] == "evan"
+
+
+def test_deep_review_capture_failure_does_not_change_result(monkeypatch):
+    """#730: a capture failure (GH 5xx or DDB error) must be swallowed -
+    the deep review already posted, so dispatch's outer result is unaffected."""
+    deep_llm = _deep_llm_with_finding()
+    tier1_llm = LlmReviewResponse(
+        kind="reviewed", findings=(), backend_used=Backend.POOLSIDE,
+        model_name="laguna",
+    )
+    monkeypatch.setattr(cr_dispatch, "review_diff", lambda *a, **kw: tier1_llm)
```

Ruff (0.15.21)

[warning] Unused lambda argument: `a`

(ARG005)